openmetadata-ui/src/main/resources/ui/cypress/common/common.js

```diff
@@ -1014,10 +1014,20 @@ export const updateOwner = () => {
       cy.get('[data-testid="hiden-layer"]').should('exist').click();
       interceptURL('GET', '/api/v1/users?limit=15', 'getUsers');
       // Clicking on edit owner button
-      cy.get('[data-testid="add-user"]').should('be.visible').click();
+      cy.get('[data-testid="edit-owner"]').should('be.visible').click();
+
+      cy.get('.user-team-select-popover')
+        .contains('Users')
+        .should('exist')
+        .should('be.visible')
+        .click();
 
       cy.get('[data-testid="selectable-list"]')
+        .eq(1)
+        .should('exist')
+        .should('be.visible')
         .find(`[title="${text.trim()}"]`)
+        .should('be.visible')
```

Comment:
we are avoiding should('be.visible') if next command is for interaction like click or something